front/lib/api/tables.ts

```diff
@@ -431,3 +440,29 @@ async function rowsFromCsv(
 
   return new Ok(rows);
 }
+
+function getSanitizedHeader(rawHeader: string[]) {
+  return rawHeader.reduce<string[]>((acc, curr) => {
+    const slugifiedName = slugify(curr);
+
+    if (!acc.includes(slugifiedName)) {
+      acc.push(slugifiedName);
+    } else {
+      let conflictResolved = false;
+      for (let i = 2; i < 10000; i++) {
+        if (!acc.includes(slugify(`${slugifiedName}_${i}`))) {
+          acc.push(slugify(`${slugifiedName}_${i}`));
+          conflictResolved = true;
+          break;
+        }
+      }
```

Comment:
We could use a Map here to keep the current increment for each slugified header.

Reply:
My opinion here is that it wouldn't make the code more performant in any significant way and would make it overall harder to read. WDYT ?

Reply:
reduced to 64